lib/galaxy/datatypes/binary.py

```diff
@@ -1452,6 +1459,77 @@ def display_peek(self, dataset):
             return "MCool (HDF5) file (%s)." % (nice_size(dataset.get_size()))
 
 
+class H5_Model(H5):
```

Comment:
I think ``H5Model`` would be a more consistent class name.

Reply:
Thanks. Will update it once the name is finalized.